--- configure
+++ configure
@@ -11734,6 +11734,12 @@ if test "x$ac_cv_func__NSGetExecutablePath" = xyes
 then :
   printf "%s\n" "#define HAVE__NSGETEXECUTABLEPATH 1" >>confdefs.h
 
+fi
+ac_fn_c_check_func "$LINENO" "posix_spawnp" "ac_cv_func_posix_spawnp"
+if test "x$ac_cv_func_posix_spawnp" = xyes
+then :
+  printf "%s\n" "#define HAVE_POSIX_SPAWNP 1" >>confdefs.h
+
 fi
 ac_fn_c_check_func "$LINENO" "mkstemp" "ac_cv_func_mkstemp"
 if test "x$ac_cv_func_mkstemp" = xyes
